src/libstd/primitive_docs.rs

```diff
@@ -710,6 +710,10 @@ mod prim_u128 { }
 //
 /// The pointer-sized signed integer type.
 ///
+/// The size of this primitive is how many bytes it takes to reference any
+/// one byte in memory. For example, on a 32 bit target, this is 4 bytes
```

Comment:
nit: "one byte" here distracted me, though perhaps only in context of reviewing something about sizes.  Perhaps a phrasing like "to reference any location in memory"?

Reply:
If I change it to this, I have to also describe what a location in memory is, no?

Reply:
Hmm, `*T::offset` talks of "allocated object", so maybe use that instead of location?  Though I don't know if it's defined anywhere (other than LLVM) either...